aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -49,7 +49,7 @@ module aptos_framework::object {
     /// Cannot reclaim objects that weren't burnt.
     const EOBJECT_NOT_BURNT: u64 = 8;
     /// Object is untransferable any operations that might result in a transfer are disallowed.
-    const ENOT_MOVABLE: u64 = 9;
+    const EOBJECT_NOT_TRANSFERRABLE: u64 = 9;
```

Comment:
nit: the error message/comment needs a dot or a comma? "The object is untransferable, any ..."